test/e2e/e2e.go

```diff
@@ -459,30 +486,24 @@ var _ = ginkgo.Describe("e2e control plane", func() {
 		podChan, errChan := make(chan *v1.Pod), make(chan error)
 		go func() {
 			defer ginkgo.GinkgoRecover()
-			checkContinuousConnectivity(f, "", "connectivity-test-continuous", "8.8.8.8", 53, 30, podChan, errChan)
+			checkContinuousConnectivity(f, "", "connectivity-test-continuous", "8.8.8.8", 53, 30, 30, podChan, errChan)
 		}()
 
 		testPod := <-podChan
-		framework.Logf("Test pod running on %q", testPod.Spec.NodeName)
-
-		time.Sleep(5 * time.Second)
-
-		podClient := f.ClientSet.CoreV1().Pods("ovn-kubernetes")
+		nodeName := testPod.Spec.NodeName
+		framework.Logf("Test pod running on %q", nodeName)
 
-		podList, _ := podClient.List(context.Background(), metav1.ListOptions{})
-		podName := ""
-		for _, pod := range podList.Items {
-			if strings.HasPrefix(pod.Name, "ovnkube-node") && pod.Spec.NodeName == testPod.Spec.NodeName {
-				podName = pod.Name
-				break
-			}
-		}
+		ginkgo.By("Deleting ovn-kube pod on node " + nodeName)
+		err := restartOVNKubeNodePod(f.ClientSet, "ovn-kubernetes", nodeName)
+		framework.ExpectNoError(err)
 
-		err := podClient.Delete(context.Background(), podName, metav1.DeleteOptions{})
-		framework.ExpectNoError(err, "should delete ovnkube-node pod")
-		framework.Logf("Deleted ovnkube-node %q", podName)
+		ginkgo.By("Verifying that the cluster is healthy")
+		err = waitClusterHealthy(f, numMasters)
```

Comment:
Since this is shared by many tests, would it make sense to move it to AfterEach ?

Reply:
done